lib/index.js

```diff
@@ -60,6 +34,11 @@ Chartbeat.prototype.loaded = function() {
   return !!window.pSUPERFLY;
 };
 
+Chartbeat.prototype.initialize = function() {
+  this.pageCalledYet = false; 
+  this._ready = true; // temporarily switch ready to true so that a single page call can fire
```

Comment:
I'm a little bit worried here that we're setting ready to true. In most cases we use this so the user can call some method on the global variable, e.g.

```
analytics.ready(function() {
  window._cbq.push([/* whatever */]) 
});
```

Are we sure that won't be the case here? We could also just set up this queue for them I suppose, just want to double check (and maybe leave a note)

Reply:
See other comment for reason for flipping _ready